lib/fake-xhr/index.js

```diff
@@ -1,7 +1,6 @@
 "use strict";
 
-var TextEncoder = require("text-encoding").TextEncoder;
-
+var TextEncoder = global.TextEncoder || require("../../vendor/text-encoding").TextEncoder;
```

Comment:
Instead of doing this everywhere, just do this test once in the export in `text-encoding/index.js`.